README.adoc

```diff
@@ -46,17 +44,17 @@ If you have administrative access to the general `openshift` namespace and want
 oc replace -n openshift --force -f ...
 ----
 
-See the OpenShift documentation at https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.4.Beta/html/ for more details on importing templates.
+See the OpenShift documentation at https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.4/html/ for more details on importing templates.
 
 #### Updating Existing Images
 To update an to the most recent image:
 
 [source, bash]
 ----
-oc import-image jboss-eap74-beta-openjdk8-openshift:7.4.0.Beta
-oc import-image jboss-eap74-beta-openjdk8-runtime-openshift:7.4.0.Beta
-oc import-image jboss-eap74-beta-openjdk11-openshift:7.4.0.Beta
-oc import-image jboss-eap74-beta-openjdk11-runtime-openshift:7.4.0.Beta
+oc import-image jboss-eap74-openjdk8-openshift:7.4.0
```

Comment:
shouldn't we just use the `latest` tag to update the most recent image. Once we start producing the CPs, these instructions will be out of date

Reply:
+1, I'll open another PR for that.